error/errors.go

```diff
@@ -0,0 +1,78 @@
+package errors
+
+/*
+	Custom message made in case of undefined error.
+*/
+const DEFAULT_MESSAGE = "SOMETHING WENT TERRIBLY WRONG"
+
+/*
+	Custom error made for wrapping CSV handling errors.
+*/
+type CSVFormatError struct {
+	Message string
+	Err     error
+}
+
+/*
+	Custom error made in case the searched item is not found.
+*/
+type EntityNotFoundError struct {
+	Message string
+	Err     error
+}
+
+/*
+	Custom error that wraps the errors made on request.
+*/
+type InvalidRequestError struct {
+	Message string
+	Err     error
+}
+
+/*
+	Custom error made in case any business rule is not met.
+*/
+type DomainValidationError struct {
+	Message string
+	Err     error
+}
```

Comment:
Instead of `FooBarError`, errors should be in the form `ErrFooBar`.

Remember to change the documentation with the `//` form.